src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -390,11 +388,11 @@ public async Task CreateAsync(ContentItem contentItem, VersionOptions options)
             // invoke handlers to add information to persistent stores
             await Handlers.InvokeAsync((handler, context) => handler.CreatingAsync(context), context, _logger);
 
-            await ReversedHandlers.InvokeAsync((handler, context) => handler.CreatedAsync(context), context, _logger);
-
             _session.Save(contentItem);
```

Comment:
Before `CreatedAsync()` handlers as it has not been queried, so not yet in the identity map